boxes/npx.js

```diff
@@ -43,6 +58,70 @@ function updatePathEnvVar() {
   info(`Added ${targetDir} to PATH in ${shellProfile}.`);
 }
 
+export function prettyPrintNargoToml(config) {
+  const withoutDependencies = Object.fromEntries(
+    Object.entries(config).filter(([key]) => key !== "dependencies"),
+  );
+
+  const partialToml = stringify(withoutDependencies);
+  const dependenciesToml = Object.entries(config.dependencies).map(
+    ([name, dep]) => {
+      const depToml = stringify.value(dep);
+      return `${name} = ${depToml}`;
+    },
+  );
+
+  return (
+    partialToml + "\n[dependencies]\n" + dependenciesToml.join("\n") + "\n"
+  );
+}
+
+async function replacePaths(rootDir) {
+  const files = fs.readdirSync(path.resolve(".", rootDir), {
+    withFileTypes: true,
+  });
+
+  files.forEach((file) => {
+    const filePath = path.join(rootDir, file.name);
+    // console.log(filePath);
+    if (file.isDirectory()) {
+      replacePaths(filePath); // Recursively search subdirectories
+    } else if (file.name === "Nargo.toml") {
+      let content = parse(fs.readFileSync(filePath, "utf8"));
+
+      try {
+        Object.keys(content.dependencies).forEach((dep) => {
+          const directory = content.dependencies[dep].path.replace(
+            "../../../../",
+            "",
+          );
+          content.dependencies[dep] = {
+            git: "https://github.com/AztecProtocol/aztec-packages/",
+            tag: `aztec-packages-v${version}`,
+            directory,
+          };
+        });
+      } catch (e) {
+        console.log("No Noir dependencies to update");
+      }
+
+      fs.writeFileSync(filePath, prettyPrintNargoToml(content), "utf8");
+    } else if (file.name === "package.json") {
+      try {
+        let content = JSON.parse(fs.readFileSync(filePath, "utf8"));
+        content.dependencies = {
+          ...content.dependencies,
+          "@aztec/accounts": `${version}`,
+          "@aztec/aztec.js": `${version}`,
+        };
```

Comment:
I think it'd be more robust if we searched all `@aztec/*` dependencies and replaced them, so if we add another aztec package in the future we won't have to remember updating it here.

Reply:
Definitely, I'll change this